docs/tutorials.txt

```diff
@@ -11,4 +11,4 @@ Tutorials
 
   tutorials/getting-started-sinatra
   tutorials/getting-started-rails
-  tutorials/documents
+  tutorials/mongoid-documents
```

Comment:
I do not understand this change either, can you explain please?

Reply:
You were correct. Resolved